docs/code-reviews.md

```diff
@@ -1,4 +1,67 @@
+
+![image](https://user-images.githubusercontent.com/25401512/109499428-cdb2f680-7a8c-11eb-9d77-4e53f7f1c57c.png)
+
+In this screenshot, the user is making a new Pull Request. In the top right-hand corner (marked by the 
+red box), there is a tagging function under Reviewers where you can search or select collaborators to do a code review. 
+Once you have tagged them and made the Pull Request, the person will receive a notification to their 
+email, and it will also come up in the Pull Request section of their Github main screen. 
+
+![image](https://user-images.githubusercontent.com/25401512/109499512-e91e0180-7a8c-11eb-8f7a-5024f7753366.png)
+
+It is strongly encouraged to add some information to 
+your request to aid the reviewer. Thinking about the purpose of the review can be helpful as this may 
+vary from review to review. Naming the pull request something informative and useful is a good start, 
+and even better is filling in the textbox of the Pull Request with information about what you are 
+seeking. For example, are you looking for a check of the implementation of the logic or methods 
+outlined in the protocol (in which case, linking the protocol can aid this), or are you looking for a 
+review of how well you document your code and if it is clear and understandable? Being transparent 
+will help your reviewer do a better job quicker. 
+ 
+## When to ask for Code Review?
+This question's answer will vary from user to user, but it is most useful with every change to the code. In fact, when we surveyed the OpenSAFELY team, most people reported they would like reviews to be as frequent as possible. A good way of conceptualising when to ask for a review is to think about commits and branches. A user makes a branch to change a feature or adding some new analysis, and while making that change, they do several commits to the new branch. When the time comes to merge this back to master, there is a natural inflexion point to do a code review. When you are making a Pull Request, you can tag a reviewer easily. 
+
+Merging a new branch should not be your only trigger for a review. If you are stuck or unsure what direction to take, a code review can clarify what path to take. In general, the more frequent the reviews, the better quality the code tends to be. 
+
+## How can I be a good review-ee?
+Experiences of reviewing were gathered during the code review survey, and some best practices are listed below. 
+
+- Good descriptive names for Pull Requests and commits help the reviewer follow the logic of the code and the changes made (see example of commits below)
+- A clear description of the expectations of the review is helpful, for example, are you expected to run the code or just review it in the browser?
+- Multiple smaller Pull Requests are easier than one large 
+- Code comments and documentations make the logic easier to follow 
+- Naming the order of the files, for example, `01_Data_Extraction.py`, `02_Data_Cleaning.py`, etc. 
+
```

Comment:
These are great suggestions but to some degree duplicate stuff further up the page ("Thinking about the purpose of the review can be helpful as this may  vary from review to review. Naming the pull request something informative and useful is a good start", etc).  I'd combine these two sections into one, either here, or higher up the page